Tools/c-analyzer/c_parser/parser/_regexes.py

```diff
@@ -121,6 +122,11 @@ def _ind(text, level=1, edges='both'):
             |
             (?: signed | unsigned )  # implies int
             |
+            (?:
+                (?: (?: float | double | long\s+double ) \s+ )?
+                _Complex
```

Comment:
Not sure if the regex catches it but `_Complex double` is also correct (IIRC).

Reply:
Yep, that seems working (clang/gcc), but C11 says "There are three complex types, designated as float _Complex, double _Complex, and long double _Complex."

Reply:
Actually, I took this observation from https://gcc.gnu.org/onlinedocs/gcc-4.1.2/gcc/Complex.html where the example is `_Complex double`. Formally, `_Complex` is a [type specifier](https://en.cppreference.com/w/c/language/declarations) and specifiers are allowed in arbitrary order.